WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptViewController.swift

```diff
@@ -85,11 +85,11 @@ final class ReceiptViewController: UIViewController, WKNavigationDelegate, UIPri
         guard let _ = URL(string: viewModel.receiptURLString) else {
             return
         }
-        let printController = UIPrintInteractionController.shared
         let printInfo = UIPrintInfo(dictionary: nil)
         let formattedJobName = viewModel.formattedReceiptJobName(printInfo.jobName)
         printInfo.jobName = formattedJobName
         printInfo.orientation = .portrait
+        configurePrintController()
```

Comment:
Since we now configure the print controller each time each time it is used, do we still need to also call this in `viewDidLoad()`?

Reply:
Good point